R/adjust_intensity.R

```diff
@@ -1,43 +1,85 @@
-adjust_intensity <- function(intensity, int_impro){
+#' Adjust energy intensity on technology level based on techSen specific annual improvement rates and given start and end year
+#'
+#' @param intensity energy intensity raw data in EJ/pkm or EJ/tkm
+#' @param intImproTab table that provides techScen specific annual improvement rates, start year and end year
+#' @param years time horizon of EDGE-T
+#' @return Adjusted energy intensities in EJ/pkm or EJ/tkm
 
-    ## Adjust tech mixes
-    apply_logistic_trends <- function(yrs, final, ysymm, speed, initial = 1){
-      fct <- exp((yrs - ysymm)/speed)/(exp((yrs - ysymm)/speed) + 1)
-      initial + fct * (final - initial)
-    }
 
+toolAdjust_intensity <- function(intensity, intImproTab, years){
```

Comment:
I used camelCase for all other function names... if it is not a big deal I'd opt for `toolAdjustIntensity`